test/test_typing.py

```diff
@@ -479,7 +490,7 @@ class TestCommandDocumentType(unittest.TestCase):
     @only_type_check
     def test_default(self) -> None:
         client: MongoClient = MongoClient()
-        result: Dict = client.admin.command("ping")
+        result: MutableMapping[str, Any] = cast(MutableMapping, client.admin.command("ping"))
```

Comment:
This highlights the problem with returning Mapping, it will be annoying for users to have to call cast().